addons/l10n_ch/wizard/qr_batch_error.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api, _
+
+
+class QrBatchError(models.TransientModel):
+    '''
+    Wizard :
+    When multiple invoices are selected to be printed in the QR-Iban format,
+    this wizard will appear if one or more invoice(s) could not be printed (wrong format...)
+    The user will then be able to print the valid invoices or to see a list of the non-valid invoices.
+    The non-valid invoices will have a note logged in their chatter, detailing the reason of the failure.
+    '''
+    _name = 'qr.batch.error'
+    _description = 'Handles problems occurring while creating multiple QR-invoices at once'
+
+    nb_valid_invoices = fields.Integer(default=lambda self: len(self.env.context.get('default_valid_invoices_ids')))
+    nb_faulty_invoices = fields.Integer(default=lambda self: len(self.env.context.get('default_faulty_invoices_ids')))
+    valid_invoices_text = fields.Text()
+    faulty_invoices_text = fields.Text()
+
+    @api.model
+    def default_get(self, fields):
+        rec = super().default_get(fields)
+
+        valid_len = len(self.env.context.get('default_valid_invoices_ids'))
+        faulty_len = len(self.env.context.get('default_faulty_invoices_ids'))
+
+        valid_invoices_text = str(valid_len)+" invoices are valid. Press Print to get them." if valid_len > 1 else "One invoice is valid. Press print to get it."
+        faulty_invoices_text = str(faulty_len) + " invoices are invalid." if faulty_len > 1 else "One invoice is invalid."
+        rec['valid_invoices_text'] = valid_invoices_text
```

Comment:
need a `if 'valid_invoices_text' in fields and not 'valid_invoices_text' in res`